src/popup/vault/groupings.component.html

```diff
@@ -23,131 +23,135 @@
-                    (click)="selectCollection(c.node)">
-                    <div class="row-main">
-                        <div class="icon"><i class="fa fa-fw fa-lg fa-cube"></i></div>
-                        <span class="text">{{c.node.name}}</span>
-                    </div>
-                    <span class="row-sub-label">{{collectionCounts.get(c.node.id) || 0}}</span>
-                    <span><i class="fa fa-chevron-right fa-lg row-sub-icon"></i></span>
-                </a>
+                        <span class="row-sub-label">{{typeCounts.get(cipherType.Login) || 0}}</span>
+                        <span><i class="fa fa-chevron-right fa-lg row-sub-icon"></i></span>
+                    </a>
+                    <a href="#" class="box-content-row" appStopClick appBlurClick (click)="selectType(cipherType.Card)">
+                        <div class="row-main">
+                            <div class="icon"><i class="fa fa-fw fa-lg fa-credit-card"></i></div>
+                            <span class="text">{{'typeCard' | i18n}}</span>
+                        </div>
+                        <span class="row-sub-label">{{typeCounts.get(cipherType.Card) || 0}}</span>
+                        <span><i class="fa fa-chevron-right fa-lg row-sub-icon"></i></span>
+                    </a>
+                    <a href="#" class="box-content-row" appStopClick appBlurClick (click)="selectType(cipherType.Identity)">
+                        <div class="row-main">
+                            <div class="icon"><i class="fa fa-fw fa-lg fa-id-card-o"></i></div>
+                            <span class="text">{{'typeIdentity' | i18n}}</span>
+                        </div>
+                        <span class="row-sub-label">{{typeCounts.get(cipherType.Identity) || 0}}</span>
+                        <span><i class="fa fa-chevron-right fa-lg row-sub-icon"></i></span>
+                    </a>
+                    <a href="#" class="box-content-row" appStopClick appBlurClick
+                        (click)="selectType(cipherType.SecureNote)">
+                        <div class="row-main">
+                            <div class="icon"><i class="fa fa-fw fa-lg fa-sticky-note-o"></i></div>
+                            <span class="text">{{'typeSecureNote' | i18n}}</span>
+                        </div>
+                        <span class="row-sub-label">{{typeCounts.get(cipherType.SecureNote) || 0}}</span>
+                        <span><i class="fa fa-chevron-right fa-lg row-sub-icon"></i></span>
+                    </a>
+                </div>
             </div>
-        </div>
-        <div class="box list" *ngIf="showNoFolderCiphers">
-            <div class="box-header">
-                {{'noneFolder' | i18n}}
-                <div class="flex-right">{{noFolderCiphers.length}}</div>
+            <div class="box list" *ngIf="nestedFolders && nestedFolders.length">
```

Comment:
we can use the safe navigation operator (?) to make it simple `*ngIf="nestedFolders?.length"`

Reply:
Actually, this should not be using falsey logic but explicit logic.

```suggestion
            <div class="box list" *ngIf="nestedFolders != null && nestedFolders.length > 0">
```

Reply:
Good point